aten/src/ATen/native/ChanelShuffle.cpp

```diff
@@ -28,6 +29,7 @@ Tensor channel_shuffle(const Tensor& self, int64_t groups) {
               "channel_shuffle expects input with > 2 dims, but got input with sizes ",
               self.sizes());
   int64_t c = self.size(1);
+  TORCH_CHECK(self.numel() != 0, "Input Tensor should have no zeros");
```

Comment:
Just adding it here should be sufficient as we call `native_channel_shuffle` below which dispatches to 
`channel_shuffle_cpu` or `math_channel_shuffle`
https://github.com/pytorch/pytorch/blob/4b439efb6f104e0d858eee0661cefb905b6dd505/aten/src/ATen/native/ChanelShuffle.cpp#L47

https://github.com/pytorch/pytorch/blob/4b439efb6f104e0d858eee0661cefb905b6dd505/aten/src/ATen/native/native_functions.yaml#L3482-L3485